inlong-sort/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/multiple/IcebergMultipleStreamWriter.java

```diff
@@ -130,12 +129,6 @@ public void processElement(RecordWithSchema recordWithSchema) throws Exception {
             List<Integer> equalityFieldIds = recordWithSchema.getPrimaryKeys().stream()
                     .map(pk -> recordWithSchema.getSchema().findField(pk).fieldId())
                     .collect(Collectors.toList());
-            // if physical primary key not exist, put all field to logical primary key
-            if (equalityFieldIds.isEmpty()) {

```

Comment:
It is recommend to add a option 'sink.multiple.handle-strategy-without-pk'(The value maybe ['append'|'all-columns']) to decide what to do, if it is 'append',  all the records will be write use append mode, else use upsert mode by all columns.

Reply:
good proposal